SteakBot.Core/Assets/memeCommands.json

```diff
@@ -316,5 +316,155 @@
+        "Description": "let's see what happens"
+    },
+    {
+        "ResultType": 3,
+        "Name": "ooh",
+        "Value": "https://bigcdn.funnyjunk.com/comments/+_ca9959baf3144e3f1c772ab6198673a4.mp4",
+        "Description": "O0o0o0oh!"
+    },
+    {
+        "ResultType": 3,
+        "Name": "hahaFuckYou",
+        "Value": "https://1eu.funnyjunk.com/comments/Your+newest+reaction+webm+sticky+this+bad+boi+op+fresh+_2f408377f2cef14f502d97e2375125ad.mp4",
+        "Description": "Schwarzenegger bottle laugh"
+    },
+    {
+        "ResultType": 3,
+        "Name": "mmmHaha",
+        "Value": "https://2eu.funnyjunk.com/comments/+_15c0e1a6f798070.mp4",
+        "Description": "Trying not to laugh fail"
+    },
+    {
+        "ResultType": 3,
+        "Name": "kekw",
+        "Value": "https://funnymemes.funnyjunk.com/comments/Quotnasty+ass+vacuum+erquot+_e059dbda2be77911797b8fc1d359a9ed.mp4",
+        "Description": "ETU"
+    },
+    {
+        "ResultType": 3,
+        "Name": "wooWee",
+        "Value": "https://2eu.funnyjunk.com/comments/Ok+_cb8c3a9d158eebd1da7c414a47ce6ef5.mp4",
+        "Description": "Cowboy Curtis"
+    },
+    {
+        "ResultType": 3,
+        "Name": "weebNice",
+        "Value": "https://bigcdn.funnyjunk.com/comments/+_e1275a03b920332a1425d0bca2327f32.mp4",
+        "Description": "NICENICENICE"
+    },
+    {
+        "ResultType": 3,
+        "Name": "weebNice2",
+        "Value": "https://bigcdn.funnyjunk.com/comments/+_e8ae79ab7b41339dde414af249f36c09.mp4",
+        "Description": "NICENICENICENICENICE"
+    },
+    {
+        "ResultType": 3,
+        "Name": "headsBackLaugh",
+        "Value": "https://bigcdn.funnyjunk.com/comments/Oh+_b6f460f25619525886159a21dccf1329.mp4",
+        "Description": "Throw our heads back and laugh"
+    },
+    {
+        "ResultType": 3,
+        "Name": "youDontWannaKnow",
+        "Value": "https://bigcdn.funnyjunk.com/comments/Have+a+sound+version+_15d6c835d47bc90.mp4",
+        "Description": "Trust me. He told me and now I'm telling you - udwn"
+    },
+    {
+        "ResultType": 2,
+        "Name": "monsterAmused",
+        "Value": "https://funnymemes.funnyjunk.com/thumbnails/comments/+_15473830fd41993fb91d0b9fc7ec1e81.jpg",
+        "Description": "Holding Monster™ amused"
+    },
+    {
+        "ResultType": 2,
+        "Name": "noPower",
+        "Value": "https://media.tenor.com/images/faf62164929af662c6700f94a94ccd24/tenor.gif",
+        "Description": "You have no power here meme gif"
+    },
+    {
+        "ResultType": 3,
+        "Name": "ohSnap",
+        "Value": "https://bigcdn.funnyjunk.com/comments/+_11210d334a4cc1ff5acba79a568733e7.mp4",
+        "Description": "Mark Meechan laugh"
+    },
+    {
+        "ResultType": 1,
+        "Name": "look",
+        "Value": "|| https://memestatic1.fjcdn.com/thumbnails/comments/+_36507121ffe51c2e0f93507a32670ea7.png ||",
+        "Description": "Made you"
+    },
```

Comment:
This can be removed as it's not working.

Reply:
Removed.